novawallet/Modules/DApp/DAppBrowser/StateMachine/MetamaskStates/DAppMetamaskBaseState.swift

```diff
@@ -75,11 +86,13 @@ class DAppMetamaskBaseState {
         }
 
         let ethereumChain = extendingMetamaskChainWithSubstrate(
-            MetamaskChain.etheremChain,
+            MetamaskChain.ethereumChain,
```

Comment:
I think we now can safely get the chain by the chain id from the data source instead of allowing to switch only to the Ethereum chain. WDYT?